jobinfo

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/env python3.6
```

Comment:
Is this strict dependency on python 3.6 required?

Reply:
Not necessarily, but I couldn't find a good way to make sure we use the system's Python version and `pynumparser` is found. Setting this to `/usr/bin/python3` somehow breaks if you load a Python3.6 module from our module tree (`/usr/bin/python3: symbol lookup error: /usr/bin/python3: undefined symbol: _Py_LegacyLocaleDetected`), and setting it to `/usr/bin/env python3` will break if you load e.g. 3.7 from the module system, because `pynumparser` is only available for the system's version (3.6).

Reply:
Ok. Then we have to leave this as it is.

Reply:
I don't really like this either (also because the spec file can install it for different OS Python versions), but this does seem to work for all combinations that I've tried, and I'm not sure how to fix this properly. We could insert all the `pynumparser` code into the `jobinfo` script, but that's a bit ugly too...

Reply:
Or maybe we could install it to `/usr/lib/python3/dist-packages` instead, which should work for all Python 3 versions (?).

Reply:
Hmm... seems like that's a Debian/Ubuntu thing, doesn't seem to work on CentOS...